runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileContent/ProcessTileContent.styles.ts

```diff
@@ -1,14 +1,19 @@
-import { CardContent } from '@mui/material';
+import { CardContent, Box } from '@mui/material';
 import styled from 'styled-components';
 
-export const StyledContent = styled(CardContent)`
+export const StyledContent = styled(CardContent)(({ theme }) => `
     display: grid;
-    grid-template-columns: 1fr 1.5fr;
-    grid-template-rows: repeat(2, auto);
-    justify-items: start;
-    justify-content: space-evenly;
+    grid-template-columns: 2fr 1.9fr;
+    gap: 0.5rem;
+    height: 100%;
     width: 100%;
+    background: linear-gradient(${theme.palette.grey[300]}, ${theme.palette.grey[300]}) center/1.5px 80% no-repeat;
```

Comment:
just read background hex color from the UI mock. This line is generated for sure

Reply:
Replaced with vertical line done with div.